tests/Model/ModelManagerTest.php

```diff
@@ -249,6 +250,10 @@ public function testGetParentMetadataForProperty()
         list($metadata, $lastPropertyName) = $modelManager
             ->getParentMetadataForProperty($containerEntityClass, 'associatedEntity.embeddedEntity.plainField');
         $this->assertEquals($metadata->fieldMappings[$lastPropertyName]['type'], 'boolean');
+
+        list($metadata, $lastPropertyName) = $modelManager
+            ->getParentMetadataForProperty($containerEntityClass, 'associatedEntity.embeddedEntity.subEmbeddedEntity.plainField');
```

Comment:
Can you break this one more time? We follow PSR-2 and your line shouldn't be longer than 120 chars

Reply:
Done